### PR TITLE
fix(AWS Local Invocation): Strip `null` envvars for local invoke

### PR DESCRIPTION
**Pull request by yyamano**
Closes: #{9261}

### CODE REVIEW

File: lib/plugins/aws/invokeLocal/index.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ class AwsInvokeLocal {
   getConfiguredEnvVars() {
     const providerEnvVars = this.serverless.service.provider.environment || {};
     const functionEnvVars = this.options.functionObj.environment || {};
-    return _.merge(providerEnvVars, functionEnvVars);
+    return _.pickBy(_.merge(providerEnvVars, functionEnvVars), (v) => v !== null);
```
Comment (pgrzesik) [inline lib/plugins/aws/invokeLocal/index.js:143]: Let's try to not use lodash-based methods if possible - we're trying to limit their use as much as possible and eventually drop that dependency

File: test/unit/lib/plugins/aws/invokeLocal/index.test.js [hunk 2]
```diff
@@ -395,6 +395,13 @@ describe('AwsInvokeLocal', () => {
       await awsInvokeLocal.loadEnvVars();
       expect(process.env.providerVar).to.be.equal('providerValueOverwritten');
     });
+
+    it('it should not pass null env vars to handler', async () => {
```
Comment (pgrzesik) [inline test/unit/lib/plugins/aws/invokeLocal/index.test.js:399]: All new tests should we written with `runServerless` utility - more details available here: https://github.com/serverless/serverless/blob/master/test/README.md 

You can take a look at this one to serve as an example on how to write a similar test for introduced fix: https://github.com/serverless/serverless/blob/7072795a50ce87879089d41335641ac37d4abd09/test/unit/lib/plugins/aws/invokeLocal/index.test.js#L1542

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thanks @yyamano - I have a few comments, please take a look at let me know what do you think :bow:

File: lib/plugins/aws/invokeLocal/index.js (hunk 1 above)
Comment (yyamano) [reply lib/plugins/aws/invokeLocal/index.js]: I did it, but it doesn't look clean to me. Is that OK with you?

File: test/unit/lib/plugins/aws/invokeLocal/index.test.js (hunk 2 above)
Comment (yyamano) [reply test/unit/lib/plugins/aws/invokeLocal/index.test.js]: I tried to rewrite the test with runServerless.

File: lib/plugins/aws/invokeLocal/index.js [hunk 3]
```diff
@@ -140,7 +140,13 @@ class AwsInvokeLocal {
   getConfiguredEnvVars() {
     const providerEnvVars = this.serverless.service.provider.environment || {};
     const functionEnvVars = this.options.functionObj.environment || {};
-    return _.merge(providerEnvVars, functionEnvVars);
+    const mergedVars = _.merge(providerEnvVars, functionEnvVars);
+    return Object.keys(mergedVars)
```
Comment (pgrzesik) [inline lib/plugins/aws/invokeLocal/index.js:144]: While a bit more verbose, I think it's fine - we can refactor it to `entries/fromEntries` after we drop support for Node 10.x.

**Review by pgrzesik — APPROVED** [review]
Thanks @yyamano, looks good :+1:

File: lib/plugins/aws/invokeLocal/index.js (hunk 1 above)
Comment (pgrzesik) [reply lib/plugins/aws/invokeLocal/index.js]: Let's keep it this way and after dropping support for Node 10.x we can refactor it with `entries/fromEntries` notation

**Pull request merged**